README.md

```diff
@@ -99,3 +99,23 @@ Instructions are the same as for dependencies (see above) with the following dif
 
 1. Directory must be created in `plugins/GEM_NAME`
 2. Plugins typically depend on `foreman` or `foreman-proxy`
+
+## HOWTO: compare packages
+
+Sometimes it is beneficial to compare the contents of this Git repository with the contents on deb.theforeman.org.
+
+```console
+$ ./scripts/compare_deb_repo.py --release X.Y
```

Comment:
This covers the usage of script however if I want to take next steps on how to build packages which are missing in deb repo or how to cleanup packages then those steps should be added too?

Reply:
Not the scope of this PR? ;)

But yeah, adding that would be nice.